test_scripts/Defects/7_0/3164_2_GetInteriorVehicleData_empty_module_data_to_cache.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/3164
+--
+-- Description: Check the successful saving of empty module data in cache
+--
+-- Precondition:
+-- 1. SDL Core and HMI are started
+-- 2. RC app is registered and activated
+-- 3. PTU with permission for RC is performed
+
+-- Steps:
+-- 1. Mobile app requests subscription for <moduleType> via GetInteriorVehicleData RPC
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(<moduleType>) to HMI
+-- 2.  HMI responds with empty moduleData to RC.GetInteriorVehicleData(<moduleType>)
+-- SDL does:
+-- a. save received moduleData from HMI in cache
+-- b. resend GetInteriorVehicleData response to mobile app
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local rc = require('user_modules/sequences/remote_control')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+-- [[ Local Variables ]]
+local appId = 1
+local isSubscribeVal = true
+local withoutSubscription = nil
+local isCachedVal = true
+local moduleData = {
+  emptyRadioData = { moduleType = "RADIO", radioControlData = {}},
+  emptyAudioData = { moduleType = "AUDIO", audioControlData = {}},
+  emptyHmiSettingsData = { moduleType = "HMI_SETTINGS", hmiSettingsControlData = {}},
+  emptyClimateData = { moduleType = "CLIMATE", climateControlData = {}},
+  emptySeatData = { moduleType = "SEAT", seatControlData = {}},
+}
+
+-- [[ Local Functions ]]
+local function getModuleId(pModuleType)
+  return rc.predefined.getModuleControlData(pModuleType, 1).moduleId
+end
+
+local function GetInteriorVehicleData(pModuleData, isSubscribe, isCached)
```

Comment:
@GetmanetsIrina 
```suggestion
local function GetInteriorVehicleData(pModuleData, pIsSubscribe, pIsCached)
```

This comment relates to other scripts of the PR too.